libs/web/src/utils/constants.ts

```diff
@@ -62,13 +62,13 @@ export const BASE_ASSETS = [ETH_ASSET_ID, USDC_ASSET_ID];
 export const DisclaimerMessage = `Disclaimer
 1. I am not a person or entity who resides in, is a citizen of, is incorporated in, or has a registered office in the United States of America or any other Prohibited Localities, as defined in the Terms of Use.
 
-2. I will not access this site or use the Mira Dex protocol while located within the United States or any Prohibited Localities.
+2. I will not access this site or use the Microchain protocol while located within the United States or any Prohibited Localities.
```

Comment:
DEX is Decentralized Exchange
```suggestion
2. I will not access this site or use the Microchain Dex protocol while located within the United States or any Prohibited Localities.
```

Do we need to remove that?